core/translate/expr.rs

```diff
@@ -1841,6 +1841,33 @@ pub fn translate_expr(
                                 "DETACH should be handled at statement level, not as expression"
                             );
                         }
+                        ScalarFunc::Unlikely => {
```

Comment:
`likely` and `unlikely` shouldn't even call any function, they should just translate the argument as is -- the existing implementation of `exec_likelihood` etc. are all wrong. I think we could fix them here.